index.html

```diff
@@ -30,19 +31,59 @@
     <br />
     <br />
     <div class="bttn-container">
-      <button class="bttn-1">
+      <button class="bttn-1" id="about-bttn">
         <div class="overlap-group">
           <div class="buttn-1-text">sobre</div>
-          <!-- <div class="popup" onclick="myFunction()">
-            <span class="popuptext" id="myPopup">Popup text...</span> -->
         </div>
       </button>
+      <div id="aboutModal" class="modal">
```

Comment:
Let's keep the attribute names consistent with kebab-case 🤗 

```suggestion
      <div id="about-modal" class="modal">
```